### PR TITLE
Added the folder for Translator and the Open API file for Text Translation API V3

### PR DESCRIPTION
**Pull request by kellyaltom**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
The empty README.md files are causing various processes to fail and need to be fixed.

File: specification/cognitiveservices/data-plane/TextTranslation/V3/TextTranslationV3.json [hunk 1]
```diff
@@ -0,0 +1,930 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "3.0",
+    "title": "Translator Text API",
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/TextTranslation/V3/TextTranslationV3.json]: Title will be used for client class name
Please change to "Translator Text Client" to keep consistent with other Cognitive Services.

**Comment by kellyaltom** [comment]
Empty readme files causing problems. Fixing.

**Comment by kellyaltom** [comment]
done


________________________________

File: specification/cognitiveservices/data-plane/TextTranslation/V3/TextTranslationV3.json [hunk 2]
```diff
@@ -0,0 +1,930 @@
+{
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/TextTranslation/V3/TextTranslationV3.json]: I noticed some places we call the service "Translator Text" some places we call it "Text Translation".
We should use same term.

If "Translator Text" is the official service name, in here, the path should be:
specification/cognitiveservices/data-plane/TranslatorText/v3.0/TranslatorText.json
```
TextTranslation => TranslatorText
V3 => v3.0
TextTranslationV3.json => TranslatorText.json
```

File: specification/cognitiveservices/data-plane/TextTranslation/Readme.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Microsoft Translator API V3
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/TextTranslation/Readme.md]: Readme.md => readme.md :)
This file is not a simply readme file. This is actually an configuration/metadata file for autorest.

Please follow the examples in
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/Face/readme.md 
to write an autorest compatible readme.

Comment (jhendrixMSFT) [reply specification/cognitiveservices/data-plane/TextTranslation/Readme.md]: Yes please be sure to use lower-case in the file name.  Also the file format is incorrect and is still causing breaks, please see the logs for details.

**Comment by kellyaltom** [comment]
Made the fixes to the Open API file and the Readme.md files.

File: specification/cognitiveservices/data-plane/TranslatorText/Readme.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Microsoft Translator API V3
```
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/TranslatorText/Readme.md]: You'll need to rename this file to something else as ```readme.md``` has special meaning in this repo.  Or just remove it entirely as I don't know how much value it provides regarding swagger generation.

**Comment by kellyaltom** [comment]
OK


I thought the customers would just try the operation out to see what a successful request looks like.


A successful request of the Languages method will return 15 screens of data. Do you want all of the data from a successful request or just a subset.


________________________________

**Comment by jhendrixMSFT** [comment]
The examples don't have to be huge, just enough to convey what a request and response can look like.  Building on my previous example link, see the details of an example file [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/SpellCheck/stable/v1.0/examples/SuccessfulProofModeRequest.json).

**Comment by kellyaltom** [comment]
I plan on getting back to it late next week.

We received a lot of last minute requests from SOC auditors that needed to be done this month, and we have a conference next week where I need to give a hands on lab.


Should I cancel the pull request and start a new one?


________________________________

**Comment by jhendrixMSFT** [comment]
If you think the work is roughly a week out yeah let's close it for now, you can reopen it when ready or create a new one (whatever works best for you).

**Pull request closed without merging**